src/editor/components/monaco-language-init.js

```diff
@@ -0,0 +1,39 @@
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api";
+
+import {
+  language as cssLang,
+  conf as cssConf
+} from "monaco-editor/esm/vs/basic-languages/css/css";
+import {
+  language as mdLang,
+  conf as mdConf
+} from "monaco-editor/esm/vs/basic-languages/markdown/markdown";
+import {
+  language as jsLang,
+  conf as jsConf
+} from "monaco-editor/esm/vs/basic-languages/javascript/javascript";
+import {
+  language as pyLang,
+  conf as pyConf
+} from "monaco-editor/esm/vs/basic-languages/python/python";
+
+import {
+  language as iomdLang,
+  conf as iomdConf
+} from "../iomd-tools/iomd-monaco-lang-def";
+
+Object.entries({
+  iomd: [iomdLang, iomdConf],
+  js: [jsLang, jsConf],
+  md: [mdLang, mdConf],
+  css: [cssLang, cssConf],
+  py: [pyLang, pyConf]
+}).forEach(lang => {
```

Comment:
Do we need to update the language plugin documentation for any new conventions here?